libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/dialogs/ConfirmationDialog.kt

```diff
@@ -106,6 +113,9 @@ private fun ContentToPreview() {
     ConfirmationDialog(
         title = "Title",
         content = "Content",
-        thirdButtonText = "Disable"
+        thirdButtonText = "Disable",
+        onSubmitClicked = {},
+        onDismiss = {},
+        emphasizeSubmitButton = true,
```

Comment:
Maybe add another preview with `false`. Not sure this is mandatory though. We have a `BooleanProvider` if necessary.